src/main/java/com/uid2/shared/model/Site.java

```diff
@@ -134,7 +134,7 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         Site site = (Site) o;
-        return id == site.id && name.equals(site.name) && description.equals(site.description) && enabled.equals(site.enabled) && domainNames.equals(site.domainNames) && clientTypes.equals(site.clientTypes) && visible.equals(site.visible) && created == site.created;
+        return id == site.id && name.equals(site.name) && description.equals(site.description) && enabled.equals(site.enabled) && domainNames.equals(site.domainNames) && clientTypes.equals(site.clientTypes) && Objects.equals(visible, site.visible) && created == site.created;
```

Comment:
Let's add a unit test similar to https://github.com/IABTechLab/uid2-shared/blob/master/src/test/java/com/uid2/shared/auth/KeysetTest.java#L11 to catch these kind of errors